health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/FinalPopup.js

```
@@ -0,0 +1,58 @@
+                    width: "50rem"
+                }}
+                footerChildren={[
+                    <Button
+                        type={"button"}
+                        size={"large"}
+                        variation={"secondary"}
+                        label={t("CANCEL")}
+                        onClick={() => {
+                            setShowFinalPopup(false);
+                        }}
+                    />,
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)